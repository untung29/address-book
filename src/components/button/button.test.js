import React from "react";
import renderer from "react-test-renderer";
import Button from "./button.component";

it("<Button/> Snapshot Testing", () => {
  const ButtonTest = renderer.create(<Button />).toJSON();
  expect(ButtonTest).toMatchSnapshot();
});
