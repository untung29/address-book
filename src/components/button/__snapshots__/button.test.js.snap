// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button/> Snapshot Testing 1`] = `
<button
  className="btn btn-primary  undefined"
/>
`;
